src/main/kotlin/com/sourcegraph/cody/config/ui/UpdateChannel.kt

```diff
@@ -0,0 +1,15 @@
+package com.sourcegraph.cody.config.ui
+
+import com.intellij.util.ui.PresentableEnum
+
+enum class UpdateChannel(val channelUrl: String?) : PresentableEnum {
+  Stable(null as String?),
+  Alpha("https://plugins.jetbrains.com/plugins/alpha/9682");
+
+  override fun getPresentableText(): String {
+    return when (this) {
+      Stable -> "Stable"
+      Alpha -> "Alpha"
```

Comment:
Let's call it the "Nightly" channel. We can later consider deleting the alpha channel and creating a new "Nightly" channel instead.